lib/internal/webstreams/readablestream.js

```diff
@@ -681,6 +684,14 @@ class ReadableStreamBYOBRequest {
         'This BYOB request has been invalidated');
     }
 
+    if (!isArrayBufferView(view)) {
+      throw new ERR_INVALID_ARG_TYPE(
+        'view',
+        ['Buffer', 'TypedArray', 'DataView'],
+        view,
+      );
+    }
```

Comment:
Can you use `validateBuffer` instead? https://github.com/nodejs/node/blob/d83446b4c4694322e12d2b7d22592f2be674e580/lib/internal/validators.js#L200-L206